### ISSUE TITLE
about find(). search someone out from html tag,it return blank

### ISSUE THREAD

**Issue by maoSting** [voku/simple_html_dom#50]
#### it can't find tag out from html.

```
$html    = '<!DOCTYPE HTML>
<html>
<head>
    <title>title</title>
</head>

<body>
<div id="a">
    an apple
</div>
</body>

</html>
<div id="b">
    body
</div>';
$domTree = \voku\helper\HtmlDomParser::str_get_html($html);
var_dump($domTree->findOne('#a')->text()); // an apple

var_dump($domTree->findOne('#b')->text()); // empty
```

**Comment by voku** [comment]
Your HTML is not valid. Please check it: https://validator.w3.org/nu/#textarea

PS: why do you need to add the DIV outside your HTML-tag?

**Comment by petrospap** [comment]
Hello, I have a similar question  (don't want to open new issue)
how to determine if we find something?
i.e
```
$html = HtmlDomParser::file_get_html('https://example.com');
foreach ($html->find('img#test') as $img) {
    if ($img->src) {
		echo 'found';
    }else{
		echo 'NOT found';
	}
}
```
Any idea?

**Comment by voku** [comment]
There are different find* methods, please take also a look at the tests. 😊

**Comment by petrospap** [comment]
Very helpful, although a simply isset can do the trick :)
BTW very nice work!!!
Thanks!

**Comment by maoSting** [comment]
when i spider some web sites. some gays wrote html is an suprise...

**Comment by voku** [comment]
Fixed in version v4.7.20
